scripts/route_check.py

```diff
@@ -338,6 +339,23 @@ def get_interfaces():
     return sorted(intf)
 
 
+def get_chassis_local_interfaces():
```

Comment:
This can be useful API for others too. How about adding in 'utilities_common/chassis.py'.

Reply:
@smaheshm  I have added  the  utilities_common/chassis.py as your suggestion.